cql3/functions/aggregate_fcts.cc

```diff
@@ -468,6 +494,32 @@ static shared_ptr<aggregate_function> make_count_function() {
 }
 }
 
+// Drops the first arg type from the types declaration (which denotes the accumulator)
+// in order to compute the actual type of given user-defined-aggregate (UDA)
+static std::vector<data_type> state_arg_types_to_uda_arg_types(const std::vector<data_type> &arg_types) {
```

Comment:
`&` stuck to the arg instead of the types. Maybe it was confused by the arg named "arg_types".